### PR TITLE
Lock camera and view orientation to status bar orientation

### PR DESCRIPTION
**Pull request by edwardgroberski**

### CODE REVIEW

File: Source/Helper.swift [hunk 1]
```diff
@@ -15,14 +15,13 @@ struct Helper {
...
-
+  
   static func videoOrientation() -> AVCaptureVideoOrientation {
-    switch UIDevice.current.orientation {
-    case .portrait: return .portrait
-    case .landscapeLeft: return .landscapeRight
-    case .landscapeRight: return .landscapeLeft
-    case .portraitUpsideDown: return .portraitUpsideDown
-    default: return .portrait
+    switch UIApplication.shared.statusBarOrientation {
```
Comment (edwardgroberski) [inline Source/Helper.swift:20]: We want to get the orientation that the app is in before clicking camera and lock it into that.

File: Source/ImagePickerController.swift [hunk 2]
```diff
@@ -243,11 +243,6 @@ open class ImagePickerController: UIViewController {
       selector: #selector(volumeChanged(_:)),
       name: NSNotification.Name(rawValue: "AVSystemController_SystemVolumeDidChangeNotification"),
       object: nil)
-
```
Comment (edwardgroberski) [inline Source/ImagePickerController.swift:246]: We no longer want the view to react to a change of orientation.

File: Source/ImagePickerController.swift [hunk 3]
```diff
@@ -421,7 +416,11 @@ extension ImagePickerController: CameraViewDelegate {
   // MARK: - Rotation
 
   open override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
-    return .landscape
+    if UIApplication.shared.statusBarOrientation == .landscapeLeft {
```
Comment (edwardgroberski) [inline Source/ImagePickerController.swift:419]: Support the orientation of the app

**Review by pmcavoy89 — APPROVED** [review]
LGTM

**Pull request merged**